mod.js

```diff
@@ -4,15 +4,14 @@ import sal from "./sal.js";
 
 import adapter from "./adapter.js";
 
-export default function (config) {
+export default function (config = { dir: "." }) {
   return ({
     id: "minisearch",
     port: "search",
     load: () => {
-      const dir = config.dir || ".";
-      const db = new DB(path.join(dir, "hyper-search.db"));
+      const db = new DB(path.join(config.dir, "hyper-search.db"));
 
-      window.addEventListener("unload", () => {
+      self.addEventListener("unload", () => {
```

Comment:
Will `self` work here?

Reply:
in deno 1.14 - it says not to use window, to use either self or globalThis